src/components/paint-editor/paint-editor.css

```diff
@@ -90,10 +90,6 @@ $border-radius: 0.25rem;
     overflow: visible;
 }
 
-.with-eye-dropper {
-    cursor: none;
-}
```

Comment:
Is this change relevant to this feature?

Reply:
Ah, never mind, I see it got replaced w/`.hide-cursor` in `scrollable-canvas.css`